tools/areas.go

```diff
@@ -0,0 +1,121 @@
+package tools
+
+import (
+	"bufio"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/go-errors/errors" // warning: replaces standard errors
+)
+
+type storageArea struct {
+	NumBCLines	int			`json:"Num BC Lines"`
```

Comment:
Looks like the auto-linter isn't formatting docs on save?

Reply:
Indeed it wasn't working and since Github treats tab differently than VScode, it is was obvious on GitHub but not on VSCode. 

Fixed now.